Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 members = [
-    "pg_bm25",
+    "pg_search",
```

Comment:
can we reorder here since the order is now flipped w/ pg_analytics for alphabetical ordering